Lib/test/test_importlib/test_path.py

```diff
@@ -26,6 +27,17 @@ def test_reading(self):
 class PathDiskTests(PathTests, unittest.TestCase):
     data = data01
 
+    def test_package_spec_origin_is_None(self):
```

Comment:
Probably since this package represents a package _not_ on the file system (PathDisk), it should be in a separate test case. Does the test depend at all on PathTests or PathDiskTests.data?